### PR TITLE
Update Webpack & config.

### PR DESCRIPTION
**Pull request by DFurnes**
### Changes
This pull request updates our build pipeline to use the [latest updates to our Webpack config](https://github.com/DoSomething/webpack-config/pull/5) and our newly extracted [Babel config](https://github.com/DoSomething/babel-config). It also hacks our Wercker box to have NPM 3.x because we were getting some nasty unexpected errors otherwise.

🛠

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,20 +2,17 @@
   "private": true,
   "scripts": {
     "lint": "eslint resources/assets/**/*.js || exit 0",
```
Comment (weerd) [inline package.json]: Pull down and rebase cause this changed!

File: package.json [hunk 2]
```diff
@@ -2,20 +2,17 @@
...
   },
   "babel": {
     "presets": [
-      "es2015",
-      "react"
-    ],
-    "plugins": [
-      "transform-object-rest-spread",
-      "transform-export-extensions"
+      "@dosomething/babel-config"
```
Comment (weerd) [inline package.json]: 👍

File: package.json (hunk 1 above)
Comment (DFurnes) [inline package.json]: Done!

**Comment by DFurnes** [comment]
[As a developer, I want to update our Webpack & Babel configs, and fix the strange build issues that have been occurring on Wercker.](https://trello.com/c/gP13bVyj/575-as-a-developer-i-want-to-update-our-webpack-babel-configs-and-fix-the-strange-build-issues-that-have-been-occurring-on-wercker)

**Pull request merged**